db.rb

```diff
@@ -28,6 +68,17 @@ class Reason < ActiveRecord::Base
 end
 
 class Room < ActiveRecord::Base
```

Comment:
Typically, instead of class methods, we'd define instance methods for all these things. So you can call `Room.find(3).on?` for example.